### ISSUE TITLE
[Feature] Tags Page > Tag Item > Additional Performers (Not named like that) Nav Item for Performers who have scenes with that Tag

### ISSUE THREAD

**Issue by elkorol** [stashapp/stash#4619]
**Is your feature request related to a problem? Please describe.**
On the tags page for any tag item, we currently have a performers nav bar option for performers who are tagged with that tag.

It would be nice to see an additional option that would list performers who have scenes tagged with that specific tag.

Giving it an apt name would be the hardest part, because we can't just name it performers etc.

I would see two options basically, giving this new nav bar item a two name conjunction with any other nav bar name. For example;
Scene Performers.

But giving a nav bar item two words to describe it might be too much especially reusing a word already on the nav bar. So the second option would be to use perhaps the word scene and a symbol like

[Scene 🎭] - Using the theater mask emoji to symbolize performers in scenes, or
[Scene 👥] - Utilizing a people icon to signify performers involved in scenes.

Having the option to show performers tagged with that tag is important, as it's part of the current setup, IMHO having a nav bar option for performers who's scenes are tagged with that tag would be a far more useful and a self populating view. Because performer tagging is completely manual with no tags from StashDB relying on the user to populate these tags.

While I do tag performers to a degree with not just description tags but also organisational flags, I don't do it extensively for all performers within my Stash, so for the most part for nearly all tags, the now current performer nav bar item is mostly redundant. Although I would never want to see it gone because it is useful. 

I think having this extra item would be a nice touch, hopefully people can chip in with their comments on this one.

**Comment by echo6ix** [comment]
Very good idea.

A cleaner way to implement this on the ui without crowding the tab nav-items is to add a new small button group to the performer toolbar div. The button group would function to switch between performer tags and scene tags, where the default scene icon and default tag icon would visually represent the options instead of string labels, as well as either's active or inactive state.

Basically this, but with the appropriate corresponding icons:

![image](https://github.com/stashapp/stash/assets/37937507/b3cd297c-6a2a-41a5-b743-9dd13a8ad10a)

**Comment by elkorol** [comment]
That's a good idea, so once you're already in the performers nav tab, have
it default to one of those buttons and you can switch between them. Haven't
heard any feedback yet. I had a little look to see how to do it but I think
it would require new GraphQL. I'm not sure how to nest performers, scenes
and tags in the schema.

I'll keep trying to figure it out see if I can come up with something that
works

**Comment by echo6ix** [comment]
TLDR your original idea of simply adding an extra nav-item is simpler.

But it still might add some confusion with the tag grid view card btn-popover counters, as they seem to correspond precisely to the available nav-item counters.

**Comment by elkorol** [comment]
I'm not sure where you mean, popovers on the actual scenes?

It's not a problem so much for desktop use this. But on mobile it is.

Another option could be don't create an entirely new nav entry. It could be two click to access using the same performer nav items. Perhaps when you first click on performers, then a symbol for scenes could render within the performer nav bar item. With some way to indicate that the nav item has a 3 states. Inactive and two activate states. The first for performers with tags and the second for performers with tagged scenes.

I don't know how that would go down, it might add to confusion but it would keep a clean ui. The scenes symbol on the performer button that only appears when you first click performer. Could have a popover with text saying something like "Click to view Performers who have scenes with tag".

It would be two menu items taking up one menu item. a two click process to view both of them. On click not much changes apart from the addition of a scenes symbol, the performer text stays the same. But in truth it replaces it with a new nav button with a different link.

I'm not sure if that would be intuitive.

**Comment by echo6ix** [comment]
> I'm not sure where you mean, popovers on the actual scenes?

On http://localhost:9999/tags > each card is associated with a tag > bottom of the card has icons displaying counts values, including the count of performers associated with the corresponding _performer tag_. See image below

![image](https://github.com/stashapp/stash/assets/37937507/56d4a09d-d4d1-46ac-9679-fd4660da5b16)

**Comment by elkorol** [comment]
Could potentially maybe do something like this. [Font Awesome Stacking](https://fontawesome.com/docs/web/style/stack)

Could maybe stack two icons like this
<img width="717" alt="Untitled" src="https://github.com/stashapp/stash/assets/57588838/b4fe634a-1aca-43e8-aec6-997b36c51c51">

So that the video icon is within the chalkboard icon
<img width="724" alt="ghg" src="https://github.com/stashapp/stash/assets/57588838/7f2e955a-2ddb-442a-a091-09543def105c">

What do you think?

**Comment by elkorol** [comment]
Maybe that would be too noisy for the size the icon would be, maybe just the chalkboard icon?

**Comment by echo6ix** [comment]
> Maybe that would be too noisy for the size the icon would be, maybe just the chalkboard icon?

Icon stacking is a very good idea, but you correctly point out a disadvantage

---

Another idea to play around with is only displaying the performer icon absent of any counter values. This resolves the issue of introducing another icon. Displaying the corresponding counter values would be moved to an icon popover tooltip.

Here are some examples of how this could work. Imagine the tag card, where:
- (`PERFORMER_TAG > 0` & `PERFORMER_SCENE_TAG = 0`) OR (`PERFORMER_TAG = 0` & `PERFORMER_SCENE_TAG > 0`). Display the icon ![image](https://github.com/stashapp/stash/assets/37937507/0a13c9d9-8f85-4637-ae1b-8682b2b8be6a)
- (`PERFORMER_TAG > 0` & `PERFORMER_SCENE_TAG > 0`). Display the performer icon stacked on itself to represent multiple performer icons. This indicates that there is both a _performer_tag_ and a _performer_scene_tag_

When hovering over the icon it would display more data in the tooltip. For example `9 performers` or `19 performers in scenes` or when both are true:
```
9 performers
19 performers in scenes
```

- If the icon contains only one type, clicking the icon would go directly to the corresponding page
- If the icon contains multiple types, clicking would do nothing. The tooltip popover would contain the url to the distinct sections

The disadvantage of this concept is
- Might be considered confusion to understand
- While it does not introduce noise, it displaces some info off the card and into a tooltip

**Comment by GalacticCheat** [comment]
Maista sucks cock
![Image](https://github.com/user-attachments/assets/ed12369e-cb07-434a-b49f-fbc470272035)
https://i.ibb.co/ds49xcXt/image-2-copie.jpg
